librecad/src/cmd/rs_commands.cpp

```diff
@@ -160,40 +160,49 @@ RS_Commands::RS_Commands() {
     for(auto const& [fullCmdList, aliasList, action]: g_commandList){
         //add full commands
         for(auto const& [fullCmd, cmdTranslation]: fullCmdList){
+	    // Create lowercase versions for map keys
+            QString fullCmdLower = fullCmd.toLower();
+            QString cmdTransLower = cmdTranslation.toLower();
```

Comment:
QString::toLower() uses the "C" locale. Recommended to use QLocale::toLower().

Just a suggestion. I'm not aware of usage cases when the conversions are different.